src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/client/ChipsRestClientTest.java

```diff
@@ -26,6 +26,7 @@
     private static final String CHIPS_REST_ENDPOINT = "test-endpoint";
     private static final String DATA = "data";
     private static final String CHIPS_REST_ENDPOINT_URI_VAR_PLACEHOLDER = "{chips-rest-endpoint}";
+
```

Comment:
i think this is from the merge, but no need for new line here

Reply:
Gone